lib/integration/scenarios/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -763,19 +765,23 @@ spec = describe "BYRON_TRANSACTIONS" $ do
     it "BYRON_TX_LIST_01 - Start time shouldn't be later than end time" $
         \ctx -> runResourceT @IO $ do
             w <- emptyRandomWallet ctx
-            let startTime = "2009-09-09T09:09:09Z"
-            let endTime = "2001-01-01T01:01:01Z"
+            let startTime' =
+                    parseTimeOrError False defaultTimeLocale "%Y-%m-%dT%H:%M:%SZ"
+                    "2009-09-09T09:09:09Z"
+            let endTime' =
+                    parseTimeOrError False defaultTimeLocale "%Y-%m-%dT%H:%M:%SZ"
+                    "2001-01-01T01:01:01Z"
```

Comment:
Can we not reuse `fromText @Iso8601Time` here? 🤔

Reply:
This seems to work:
```haskell
            let Right (Iso8601Time startTime') = fromText "2009-09-09T09:09:09Z"
            let Right (Iso8601Time endTime') = fromText "2001-01-01T01:01:01Z"
            let link = Link.listTransactions' @'Byron w
                    Nothing
                    (Just $ Iso8601Time startTime')
                    (Just $ Iso8601Time endTime')
                    Nothing
                    Nothing
                    Nothing
            r <- request @([ApiTransaction n]) ctx link Default Empty
            expectResponseCode HTTP.status400 r
            decodeErrorInfo r `shouldBe` StartTimeLaterThanEndTime
                (ApiErrorStartTimeLaterThanEndTime startTime' endTime')
```

Reply:
thanks! make sense!